dist/index.html

```diff
@@ -0,0 +1,324 @@
+<!DOCTYPE html>
+<html lang="en" class="page">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <title>Miami</title>
+    <link rel="icon" type="image/x-icon" href="/favicon.8003f84c.png">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com"> 
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin=""> 
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="/main.48b3db1d.css">
+    <script src="/main.d8ebb8d6.js" defer=""></script>
+  <script src="/main.48b3db1d.js"></script></head>
+  <body class="page__body">
+    <header class="page__section header">
+      <div class="container">
+        <div class="header__content">
+          <div class="header__top top-actions">
+            <a href="#menu" class="icon icon--menu"></a>
+            <a href="#home" class="logo">
+              <img class="logo__image" src="/logo.6023b87e.png" alt="Miami Condo Kings">
+            </a>
+            <a href="tel:+1 234 555-55-55" class="icon icon--phone header__call"></a>
+          </div>
+    
+            <h1 class="header__title">
+              Miami
+              <br>
+              Condo Kings
+            </h1>
+        </div>
+      </div>
+    </header>
+    <nav class="page__menu menu" id="menu">
+      <div class="container">
+        <div class="menu__content">
+          <div class="menu__top top-actions">
+            <a href="#" class="icon icon--cross"></a>
+            <a href="#home" class="logo">
+              <img class="logo__image" src="/logo.6023b87e.png" alt="Miami Condo Kings">
+            </a>
+          </div>
+
+          <ul class="menu__list">
+            <li class="menu__item">
+            <a href="#" class="menu__link">Home</a>
+            </li>
+            <li class="menu__item">
+              <a href="#about-us" class="menu__link">About us</a>
+            </li>
+            <li class="menu__item">
+              <a href="#gallery" class="menu__link">Gallery</a>
```

Comment:
checklist:
8.Pictures in Gallery should increase on hover
10. Apply :hover effect for images on page (testimonials / gallery, other sections).